src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/DependencyResolution/DefaultRegistry.cs

```diff
@@ -1,21 +1,13 @@
-﻿using FluentValidation;
-using MediatR;
-using Microsoft.AspNetCore.Builder;
-using SFA.DAS.CommitmentsV2.Configuration;
-using SFA.DAS.CommitmentsV2.DependencyResolution;
+﻿using MediatR;
 using StructureMap;
 
 namespace SFA.DAS.CommitmentsV2.Api.DependencyResolution
 {
     public class DefaultRegistry : Registry
     {
-        private const string ServiceName = "SFA.DAS.CommitmentsV2";
-
         public DefaultRegistry()
         {
             For<ServiceFactory>().Use<ServiceFactory>(ctx => ctx.GetInstance);
```

Comment:
I think Service Factory is in the MediatorRegistry so it shouldn't be needed here